tests/rptest/services/compatibility/flink_examples.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import os
+from .example_base import ExampleBase
+
+# The flink-examples root directory which is made in the
+# Dockerfile
+TESTS_DIR = os.path.join("/opt", "flink-kafka-examples")
+
+# The flink dirs
+FLINK_DIR = os.path.join("/opt", "flink-1.14.0")
+FLINK_BIN = os.path.join(FLINK_DIR, "bin")
+
+
+class FlinkWithKafka(ExampleBase):
+    def __init__(self, redpanda):
+        super(FlinkWithKafka, self).__init__(redpanda)
+
+    # The internal condition to determine if the
+    # example is successful. Returns boolean.
+    def _condition(self, line):
+        return "Job has been submitted with JobID" in line
```

Comment:
I know nothing about flink, but this seems odd as a success condition, it looks more like it's saying a job started than that it completed?